### PR TITLE
initial store, reducer, actions

### PR DESCRIPTION
**Pull request by JudithBLinder**
Initial Store, reducer and actions.

It wasn't a mistake. when i press on command it allows me to select more then one reviewer :)

### CODE REVIEW

File: client/store/actions.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const TYPES = {
+  SET_HEROES: 'SET_HEROES',
+};
+
+const setHeroes = (heroes) => ({ type: TYPES.SET_HEROES, heroes });
+
+export { TYPES, setHeroes };
```
Comment (DmytroLypovetskiy) [inline client/store/actions.js:7]: I think it's good to have a separate file with types. What do you think?

Comment (JudithBLinder) [reply client/store/actions.js:7]: I dont mind. ill separate them.

File: client/store/types.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const TYPES = {
+  SET_HEROES: 'SET_HEROES',
+};
+
+export default TYPES;
```
Comment (DmytroLypovetskiy) [inline client/store/types.js:5]: You can do just
export const SET_HEROES = 'SET_HEROES';
export const SET_HEROE = 'SET_HEROE';

so you will not need to destructure them in reducer and action :)

Comment (JudithBLinder) [reply client/store/types.js:5]: I like that you export TYPES and not many types.

**Pull request merged**